### PR TITLE
Pull all the dependent repositories for kbweb

### PR DESCRIPTION
**Pull request by jzila**
This PR is preparation for merging keybase/keybase#801. Once this and keybase/kbfs-server#75 are merged, keybase/keybase#801 should be safe to merge without breaking anything.

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 5ef011aad56a...ef76f73fd3ed introduced some visual changes on linux:
 * Changed: **unlock_folders-Single** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469132176845-5ef011aad56a-ef76f73fd3ed-linux/unlock_folders-Single-5ef011aad56a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469132176845-5ef011aad56a-ef76f73fd3ed-linux/unlock_folders-Single-ef76f73fd3ed.png)  
   unlock_folders-Single comparison

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+version: '2'
+services:
+    mysql.local:
+        image: keybaseprivate/mysql
+        restart: unless-stopped
```
Comment (chromakode) [inline docker-compose.yml]: Do we want a restart policy here? Won't that make tests less deterministic if MySQL is flaky (or is that the intent?)

Comment (jzila) [inline docker-compose.yml]: Honestly I've never seen that policy triggered. I can remove it if you prefer.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            - MYSQL_DATABASES=gregor
+        ports:
+            - "3306"
+        command: ["--innodb_print_all_deadlocks=1"]
+    kbweb.local:
+        image: keybaseprivate/kbweb
+        ports:
+            - "3000:3000"
+            - "13009"
+            # TODO: remove
```
Comment (chromakode) [inline docker-compose.yml]: :sheep: I assume this is quickly replaced in your other PRs, but probably a good idea to put more context in comments like this for future alien historians

Comment (jzila) [inline docker-compose.yml]: Yeah this will be replaced as soon as keybase/keybase#801 is in, which has already been approved.

File: Jenkinsfile [hunk 3]
```diff
@@ -3,7 +3,10 @@
 if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
     println "Skipping build because PR title contains [ci-skip]"
 } else {
-    nodeWithCleanup("ec2-fleet") {
+    nodeWithCleanup("ec2-fleet", {
+        sh 'docker rm -v $(docker ps --filter status=exited -q 2>/dev/null) 2>/dev/null || echo "No Docker containers to remove"'
```
Comment (chromakode) [inline Jenkinsfile]: Could devnulling the stderr from these command obscure an error?

File: docker-compose.yml (hunk 1 above)
Comment (chromakode) [inline docker-compose.yml]: Yeah I think it'd be a good idea to remove this and the copy lower in the file. should make things more predictable.

File: Jenkinsfile (hunk 3 above)
Comment (jzila) [inline Jenkinsfile]: Pretty standard practice: https://github.com/jfrazelle/dotfiles/blob/a7fd3df6ab423e6dd04f27727f653753453db837/.dockerfunc#L8-L11

**Comment by chromakode** [comment]
:+1:

**Pull request merged**